test/integration/narrow/helpers/other-items-sizes.test.js

```diff
@@ -621,8 +621,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 9999,
-            error: 'Quantity must be between 1-9999'
+            max: 999999,
+            error: 'Quantity must be between 1-999999'
```

Comment:
![image](https://github.com/DEFRA/ffc-grants-slurry-web/assets/15157025/9e44d6a1-8a7d-4391-85c5-460355a1119a)

agitators and aerators should be 1-9999